src/Algebra/FunctionProperties/Module/Left.agda

```diff
@@ -0,0 +1,55 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Properties of functions, such as associativity and commutativity
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Level
+open import Relation.Binary
+open import Data.Sum
+
+-- The properties are parameterised by the following "equality"
+-- relations
+
+module Algebra.FunctionProperties.Module.Left
```

Comment:
Everything in here feels more general than for `Module`. [This is good!] Thus perhaps should be put somewhere more findable?

Reply:
Yes, I had difficulty naming this file, and I would be happy to make it more generally named. My other working name was `Double` (in reference to having two carrier sets), though I don't think this sounds great.

Reply:
> Everything in here feels more general than for `Module`. [This is good!] Thus perhaps should be put somewhere more findable?

Maybe we should keep these for now, and come up with a good name once we have other structures which could share the namespace? Unless you have any good examples off the top of your head?

Reply:
Otherwise maybe try `Algebra.FunctionProperties.Left`?

Reply:
I've gone for `Algebra.FunctionProperties.(Left|Right)Action`, which I'm fairly happy with.